recipes/django_polymorphic/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.7.1" %}
+
+package:
+    name: django_polymorphic
+    version: {{ version }}
+
+source:
+    fn: django_polymorphic-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/d/django_polymorphic/django_polymorphic-{{ version }}.tar.gz
+    md5: c6999aa8aa4d5a992e2664f050a8e858
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+    preserve_egg_dir: True
+
+requirements:
+    build:
+        - python
+        - setuptools
+        #- django
+    run:
+        - python
+        #- setuptools
+        - django
+
+#test:
+    #requires:
+        #- coverage
+    #files:
+        #- runtests.py
+    #commands:
+        #- cp runtests.py $SRC_DIR && cd $SRC_DIR && coverage run --source=polymorphic runtests.py
```

Comment:
@kwilcox I was unable to make a test for this recipe. Since you have more experience with django can you take a look?

```
Traceback (most recent call last):
  File "runtests.py", line 65, in <module>
    runtests()
  File "runtests.py", line 62, in runtests
    execute_from_command_line(argv)
  File "/home/filipe/miniconda3/envs/_test/lib/python3.5/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/home/filipe/miniconda3/envs/_test/lib/python3.5/site-packages/django/core/management/__init__.py", line 327, in execute
    django.setup()
  File "/home/filipe/miniconda3/envs/_test/lib/python3.5/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/filipe/miniconda3/envs/_test/lib/python3.5/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/home/filipe/miniconda3/envs/_test/lib/python3.5/site-packages/django/apps/config.py", line 90, in create
    module = import_module(entry)
  File "/home/filipe/miniconda3/envs/_test/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 662, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/filipe/miniconda3/conda-bld/work/django_polymorphic-0.7.1/polymorphic/__init__.py", line 11, in <module>
    from .polymorphic_model import PolymorphicModel
  File "/home/filipe/miniconda3/conda-bld/work/django_polymorphic-0.7.1/polymorphic/polymorphic_model.py", line 19, in <module>
    from django.contrib.contenttypes.models import ContentType
  File "/home/filipe/miniconda3/envs/_test/lib/python3.5/site-packages/django/contrib/contenttypes/models.py", line 159, in <module>
    class ContentType(models.Model):
  File "/home/filipe/miniconda3/envs/_test/lib/python3.5/site-packages/django/db/models/base.py", line 94, in __new__
    app_config = apps.get_containing_app_config(module)
  File "/home/filipe/miniconda3/envs/_test/lib/python3.5/site-packages/django/apps/registry.py", line 239, in get_containing_app_config
    self.check_apps_ready()
  File "/home/filipe/miniconda3/envs/_test/lib/python3.5/site-packages/django/apps/registry.py", line 124, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
TESTS FAILED: django_polymorphic-0.7.1-py35_0
```